main.cc

```diff
@@ -24,7 +23,20 @@ main(int argc, char* argv[])
 	yy::kivi_parser parser{ ctx };
 	parser.parse();
 
-	std::cout << printer::print(ctx.functions());
+	if (cli::print_ast())
```

Comment:
I'm not quite sure for this checkings - isn't there a better way?

Reply:
Yup! I hadn't realised how ugly this actually is :smile: I just committed a rework.